openstack/check.py

```diff
@@ -779,14 +796,11 @@ def check(self, instance):
             project = self.get_scoped_project(instance)
 
             # Restrict monitoring to non-excluded servers
-            excluded_server_ids = self.init_config.get("exclude_server_ids", [])
-            servers = list(
-                set(self.get_servers_managed_by_hypervisor()) - set(excluded_server_ids)
-            )
+            server_ids = self.get_servers_managed_by_hypervisor(instance)
```

Comment:
I actually found a bug here.... Not sure what we should do, if fix the code to be consistent with the docs, or the docs to be consistent with the code. Basically, the issue is `exclude_server_ids` was documented as an instance-level configuration parameter, but was in fact an `init_config` level configuration element. This PR makes it instance-level, but I'm not sure what the right thing to do here is.